tests/cunit/test_perf2.c

```diff
@@ -539,8 +539,8 @@ main(int argc, char **argv)
 
     for (i = 0; i < num_io_tests; i++)
     {
-        for (r = 0; r < NUM_REARRANGERS_TO_TEST; r++)
-        /* for (r = 0; r < 1; r++) */
+        /* for (r = 0; r < NUM_REARRANGERS_TO_TEST; r++) */
```

Comment:
You haven't found a solution here?  It would really help if you could reproduce on a platform I have access to.

Reply:
Working on it... ;-)